src/Sulu/Bundle/WebsiteBundle/Resolver/StructureResolver.php

```diff
@@ -43,7 +43,7 @@ public function __construct(
         $this->extensionManager = $structureManager;
     }
 
-    public function resolve(StructureInterface $structure, bool $loadExcerpt = true)
+    public function resolve(StructureInterface $structure, bool $loadExcerpt = true, array $includedProperties = null)
```

Comment:
we could even deprecate calling this without a `includedProperties` parameter. not sure if there are a lot of usecases for that